README.md

```diff
@@ -30,11 +30,11 @@ Usage
             * Compares Course 102 milestone requirements against Student Smith's milestones
             * Grants Student Smith access to Course 102
 
-Standalone Testing
-------------------
-
-        $ ./run_tests
+Standalone Testing and Quality Check
```

Comment:
While you're in here it would be great if you could update the readme to correctly explain how milestones works -- the description above is an old approach that was not actually implemented